actionlist/app/components/dateRangePicker.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {
+  View,
+  Text,
+  StyleSheet,
+} from 'react-native';
+
+/* eslint-disable no-unused-vars */
+import { connect } from 'react-redux';
+import DatePicker from 'react-native-datepicker';
+import { dateFilterChange } from '../actions/actionItems';
+import { Colors } from '../lib/commons';
+import ContentHeader from './contentHeader';
+
+const style = StyleSheet.create({
+  timeMarkers: {
+    flex: 1,
+    marginTop: 10,
+    marginLeft: 15,
+    marginBottom: 10,
+    fontFamily: 'BentonSansBold, Arial, sans-serif',
+    fontSize: 15,
+    textAlign: 'left',
+  },
+});
+
+const datePickerStyles = {
+  dateInput: {
+    borderWidth: 0,
+    borderBottomWidth: 1,
+    padding: 0,
+    width: 15,
+  },
+};
+
+const DateRangePicker = ({ title, dateSet, onDateChanged }) => (
+  <View>
+    <ContentHeader>{title}</ContentHeader>
+    <View style={{ flexDirection: 'row' }}>
+      <Text style={style.timeMarkers}>From: </Text>
+      <DatePicker
+        style={{ flex: 1 }}
+        mode="date"
+        androidMode="spinner"
+        placeholder="select date"
+        date={dateSet.start}
+        showIcon={false}
+        format="YYYY-MM-DD"
+        minDate="2016-05-01"
```

Comment:
Are the `minDate` and `maxDate` properties optional? If so, leave them out because they seem awfully restrictive (and also filtering would stop working in two years without a code change).